hr_timesheet_sheet/models/hr_timesheet_sheet.py

```diff
@@ -167,7 +168,16 @@ def _default_employee(self):
     @api.multi
     @api.depends('date_start', 'date_end')
     def _compute_name(self):
+        locale = self.env.context.get('lang') self.env.user.lang or or 'en_US'
```

Comment:
😏

Reply:
:) who needs this commit checking before pushing? :)))